content/source/docs/enterprise/admin/agents-on-tfe.html.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: "enterprise"
+page_title: "Terraform Cloud Agents on TFE - Infrastructure Administration - Terraform Enterprise"
+---
+
+# Terraform Cloud Agents on TFE
+
+Terraform Cloud Agents allow Terraform Enterprise to communicate with isolated,
+private, or on-premises infrastructure. By deploying lightweight agents within a
+specific network segment, you can establish a simple connection between your
+environment and Terraform Enterprise which allows for provisioning operations and
+management.
+
+Terraform Cloud Agents on TFE behave identically to [Terraform Cloud Agents on TFC](https://www.terraform.io/docs/cloud/agents/index.html). Please reference this
+document for getting started steps and operational requirements. Exceptions and
+additional requirements are listed below.
+
+-> **Note:** Terraform Cloud Agents on TFE are available in TFE releases `v202109-1`
+and later.
+
+## Differences Between Agents on TFC and TFE
+
+* **No restriction on Agent or Agent Pool Count**: Terraform Enterprise does not
+place a limitation on the number of Agent Pools that can be created per organization,
+or the number of Agents that may register themselves with a given pool.
+
+* **Hostname Registration**: Terraform Cloud Agents registering with a TFE instance
+must define the TFE hostname via the `-address` CLI flag or `TFC_ADDRESS` environment
+variable when running the `tfc-agent` binary. By default, the `tfc-agent` binary
```

Comment:
I might not mention the binary explicitly like this. You will also need to define `TFC_ADDRESS` if you run the Docker image.

Reply:
Cleaned up :+1: